### PR TITLE
Users can see content

### PR DESCRIPTION
**Pull request by raomanasa**
[PT-LINK](https://www.pivotaltracker.com/story/show/170925556)

Description-
As a visitor
In order to get my news on the go
I want to have the articles shown in my app

TASKS-
- Display all articles
- Send GET request to backend to fetch all articles.

DISPLAY-
http://g.recordit.co/sfswniqWTb.gif

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import axios from 'axios';
```
Comment (mantanquzzmo) [inline src/index.jsx]: think you can remove this

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar, IonItem, IonLabel, IonList } from '@ionic/react';
+import React,{useEffect, useState} from 'react';
+import axios from 'axios';
 
 const Home = () => {
+
+  const [articles, setArticles] = useState ({})
+  let articleItems
+
+  useEffect(() => {
```
Comment (mantanquzzmo) [inline src/pages/Home.jsx]: swap the place of this and the getArticles function for easier readability

File: src/index.jsx (hunk 1 above)
Comment (raomanasa) [reply src/index.jsx]: ok

File: src/pages/Home.jsx [hunk 3]
```diff
@@ -10,13 +33,9 @@ const Home = () => {
...
       <IonContent className="ion-padding">
-        <p>
-          This is the  best mobile news app.{' '}
-          <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/">
-            Subscribe Here!
-          </a>{' '}
-          WELCOME!
-        </p>
+        <IonList>
+          {articleItems}
```
Comment (oliverochman) [inline src/pages/Home.jsx:37]: you don't want to show the truncated body of the article?

Comment (raomanasa) [reply src/pages/Home.jsx:37]: https://recordit.co/sfswniqWTb

Comment (raomanasa) [reply src/pages/Home.jsx:37]: yes. with this display i could do it without using trauncate.

File: src/pages/Home.jsx (hunk 2 above)
Comment (raomanasa) [reply src/pages/Home.jsx]: ok

**Comment by mantanquzzmo** [comment]
Well done. Looks good to me.

**Review by sundqvistjohan — APPROVED** [review]
Looks good to me, styling can be updated later when more functionality is in place

**Comment by EevanR** [comment]
LGTM

**Pull request merged**